.github/workflows/test.yml

```diff
@@ -1,8 +1,6 @@
 name: Test
 
-on:
```

Comment:
I see that it runs test both for `push` and for `pull_request`… So I think we could only use `push`.